src/test/java/org/opensearch/jobscheduler/rest/RestGetLockActionIT.java

```diff
@@ -0,0 +1,121 @@
+import org.opensearch.jobscheduler.transport.AcquireLockRequest;
+import org.opensearch.rest.RestHandler;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.test.rest.FakeRestChannel;
+import org.opensearch.test.rest.FakeRestRequest;
+
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
+public class RestGetLockActionIT extends OpenSearchTestCase {
+
+    private ClusterService clusterService;
+    private LockService lockService;
+    private RestGetLockAction getLockAction;
+    private String getLockPath;
+    private String testJobId;
+    private String testJobIndexName;
+    private long testLockDurationSeconds;
+    private String requestBody;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        this.clusterService = Mockito.mock(ClusterService.class, Mockito.RETURNS_DEEP_STUBS);
+        Mockito.when(this.clusterService.state().routingTable().hasIndex(".opendistro-job-scheduler-lock")).thenReturn(true);
+        this.lockService = new LockService(Mockito.mock(NodeClient.class), clusterService);
+        this.getLockAction = new RestGetLockAction(this.lockService);
+        this.getLockPath = String.format(Locale.ROOT, "%s/%s", JobSchedulerPlugin.JS_BASE_URI, "_lock");
+        this.testJobId = "testJobId";
+        this.testJobIndexName = "testJobIndexName";
+        this.testLockDurationSeconds = 1L;
+        this.requestBody = "{\"job_id\":\""
+            + this.testJobId
+            + "\",\"job_index_name\":\""
+            + this.testJobIndexName
+            + "\",\"lock_duration_seconds\":\""
+            + this.testLockDurationSeconds
+            + "\"}";
+    }
+
+    public void testGetNames() {
+        String name = getLockAction.getName();
+        assertEquals(RestGetLockAction.GET_LOCK_ACTION, name);
+    }
+
+    public void testGetRoutes() {
+        List<RestHandler.Route> routes = getLockAction.routes();
+        assertEquals(getLockPath, routes.get(0).getPath());
+    }
+
+    public void testAcquireLockRequest() throws IOException {
+
+        // Create AcquireLockRequest
+        AcquireLockRequest acquireLockRequest = new AcquireLockRequest(testJobId, testJobIndexName, testLockDurationSeconds);
+
+        // Generate Xcontent from request
+        XContentBuilder builder = XContentFactory.jsonBuilder();
+        builder.startObject();
+        builder.field(AcquireLockRequest.JOB_ID, acquireLockRequest.getJobId());
+        builder.field(AcquireLockRequest.JOB_INDEX_NAME, acquireLockRequest.getJobIndexName());
+        builder.field(AcquireLockRequest.LOCK_DURATION_SECONDS, acquireLockRequest.getLockDurationSeconds());
+        builder.endObject();
+
+        // Test request serde logic
+        XContentParser parser = XContentType.JSON.xContent()
+            .createParser(NamedXContentRegistry.EMPTY, LoggingDeprecationHandler.INSTANCE, TestHelpers.xContentBuilderToString(builder));
+        parser.nextToken();
```

Comment:
this isn't obvious why it's needed, and in other code I've seen this wrapped in an assertion that it's the start of the object or something like that. Might be useful here.